AppCenter/AppCenterTests/MSChannelGroupDefaultTests.m

```diff
@@ -56,7 +56,6 @@ - (void)tearDown {
   [MSDispatchTestUtil awaitAndSuspendDispatchQueue:self.sut.logsDispatchQueue];
 
   // Stop mocks.
-  [self.ingestionMock stopMocking];
```

Comment:
We shouldn't remove it.

Reply:
We've seen unexpected behaviors due to lack of this `stopMocking` a while ago and we adapted a pattern to call this explicitly whenever we need it or not.
I'm trying to remove unnecessary calls from the tests as the mock automatically calls `stopMocking` during its own deallocation. I've actually seen crashes that are caused by calling `stopMocking` after it is deallocated.

Reply:
It's exactly what I meant. it crashed if something is deallocated twice. For example because of issue with using `retainArguments`
the problem here is the OCMock still uses MRC + we have incorrect usages
removing `stopMocking` is hiding a problem, not solving

Reply:
I added a TODO comment instead. This is not required as the tests in the class are properly releasing objects but keep it for now. I know there are incorrect usages but it is mostly caused by using async queues and we can detect this in another way instead of relying on `stopMocking`.

Reply:
Changed mind. This is in `tearDown` for properties which won't be released/dealloced after each test. Need to keep it as `stopMocking` will be called when all tests are finished.